### DISCUSSION TITLE
Open file from rich traceback in Pycharm

### DISCUSSION THREAD

**Discussion by samsja** [Textualize/rich#2102] [Q&A] [answered]
Hey !

Rich traceback is awesome however it seems that we can't open the file where the exception was raised from pycharm anymore:

example 

```python
if __name__ == '__main__':
    raise ValueError
```
![Screenshot from 2022-03-24 17-53-13](https://user-images.githubusercontent.com/55492238/159969051-f414cf91-57c3-4a6b-a279-a3c48df7b6e2.png)

here the link is clickable (it is blue) and it is a really usefull shortcut for pycharm user

```python
from rich.traceback import install
if __name__ == '__main__':
    install(show_locals=False)
    raise ValueError
```
![Screenshot from 2022-03-24 17-52-48](https://user-images.githubusercontent.com/55492238/159969042-ad9cb6e6-b51c-4f23-aff5-a0cc10430dfd.png)

But with rich it is not clickable anymore :(

Any Idea ?

cc: @florian-hoenicke

**Comment by willmcgugan** [comment, marked as answer]
It's PyCharm that makes it clickable. In most terminals the link is clickable and takes you to the line number as well.

I think there was an old bug on PyCharm about this, but they haven't addressed it.

**Reply by samsja** [reply]
thank you for you answer, I will check with them then
